README.md

```diff
@@ -1,13 +1,6 @@
 # Express API Project
 
-Replace this readme with your own information about your project. 
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
-
-## The problem
-
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
```

Comment:
I would recommend you to write some short description for this project so it would be clear for anyone looking on your Github, what is this project about :) Take some inspo from week description in Notion